### PR TITLE
Transparent background + video example

### LINKED ISSUE #802: Transparent background example
Background could be animated water or something similar

Related https://github.com/mapbox/mapbox-gl-native/issues/10719 and https://github.com/mapbox/mapbox-gl-native/pull/11065

### PR DESCRIPTION
**Pull request by langsmith**
Resolves #802 . Still need to figure out why the `VideoView` isn't filling the entire screen, despite `match_parent` for width/height in XML 🤔 

ezgif com-resize

### CODE REVIEW

**Comment by langsmith** [comment]
Figured it out . The video of moving water in the background, makes it hard to record a GIF to post here. Welcome to switch to this branch and install the app on your phone.


screen shot 2018-08-10 at 10 30 04 am

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TransparentBackgroundActivity.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  }
+
+  @Override
+  public void onLowMemory() {
+    super.onLowMemory();
+    mapView.onLowMemory();
+  }
+
+  @Override
+  protected void onDestroy() {
```
Comment (osana) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TransparentBackgroundActivity.java]: Given that you start the video onCreate 
Should you stop it in onDestroy? 
I would be curious to see how it works on a tablet on Orientation change. My tablet is at home in PA though :(

Comment (langsmith) [reply MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TransparentBackgroundActivity.java]: Yea, I should stop it in `onDestroy()`. Fix made and pushed to this pr.

Still funky when in tablet landscape though 😞 


ezgif com-optimize 6

Comment (osana) [reply MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TransparentBackgroundActivity.java]: @langsmith What happens if you start in landscape? Is it ok then?

Comment (langsmith) [reply MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TransparentBackgroundActivity.java]: Made your suggestions regarding `onDestroy()`

Creation/destruction seems to be 👌 

ezgif com-resize

**Pull request merged**